.github/workflows/main.yml

```diff
@@ -231,16 +231,14 @@ jobs:
     uses: ./.github/workflows/build-linux.yml
     with:
       platform: linux-x64
-      make-target: 'static-jdk-image'
+      make-target: 'product-bundles test-bundles static-jdk-bundles'
```

Comment:
This will make us build the tests and the normal JDK twice, once here and once for the normal build-linux-x64 target. That seems like a useless waste of time and computing resources. There must be a better way of doing this.

Reply:
I haven't tried, but I think it's possible to not requiring building `product-bundles`. The `product-bundles` (regular JDK) is used for compiling the test java sources. I think it's possible to use the boot JDK. I'll try that.

Reply:
I removed `product-bundles test-bundles` as suggested. Thanks.

With your other suggestion to add `build-linux-x64` to the `needs` list for `test-linux-x64-static`, I didn't have to change to use the boot JDK for compiling the test java sources, since we download the bundles from the `build-linux-x64` as well and have a regular JDK. `JDK_FOR_COMPILE` is still set to `${{ steps.bundles.outputs.jdk-path }}`. That's actually better than using the boot JDK for `JDK_FOR_COMPILE`, since tests may use tools from the `-Dcompile.jdk` specified path. The boot JDK is not built from the current source, and using tools from the boot JDK may cause unexpected results.